api-reference/docstubs/resources/calendar.md

```diff
@@ -0,0 +1,117 @@
+|[getSchedule](../api/calendar-getschedule.md)|[scheduleInformation](../resources/scheduleinformation.md) collection|**TODO: Add Description**|
+|[allowedCalendarSharingRoles](../api/calendar-allowedcalendarsharingroles.md)|calendarRoleType collection|**TODO: Add Description**|
+|[List singleValueExtendedProperties](../api/calendar-list-singlevalueextendedproperties.md)|[singleValueLegacyExtendedProperty](../resources/singlevaluelegacyextendedproperty.md) collection|Get the singleValueLegacyExtendedProperties from the singleValueExtendedProperties navigation property.|
+|[Create singleValueExtendedProperties](../api/calendar-post-singlevalueextendedproperties.md)|[singleValueLegacyExtendedProperty](../resources/singlevaluelegacyextendedproperty.md)|Create a new singleValueExtendedProperties object.|
+|[Delete singleValueExtendedProperties](../api/calendar-delete-singlevalueextendedproperties.md)|None|Delete a singleValueExtendedProperties object.|
+|[Update singleValueExtendedProperties](../api/calendar-update-singlevalueextendedproperties.md)|[singleValueLegacyExtendedProperty](../resources/singlevaluelegacyextendedproperty.md)|Update the properties of a singleValueExtendedProperties object.|
+|[Get singleValueLegacyExtendedProperty](../api/singlevaluelegacyextendedproperty-get.md)|[singleValueLegacyExtendedProperty](../resources/singlevaluelegacyextendedproperty.md)|Read properties and relationships of a [singleValueLegacyExtendedProperty](../resources/singlevaluelegacyextendedproperty.md) object.|
+|[List multiValueExtendedProperties](../api/calendar-list-multivalueextendedproperties.md)|[multiValueLegacyExtendedProperty](../resources/multivaluelegacyextendedproperty.md) collection|Get the multiValueLegacyExtendedProperties from the multiValueExtendedProperties navigation property.|
+|[Create multiValueExtendedProperties](../api/calendar-post-multivalueextendedproperties.md)|[multiValueLegacyExtendedProperty](../resources/multivaluelegacyextendedproperty.md)|Create a new multiValueExtendedProperties object.|
+|[Delete multiValueExtendedProperties](../api/calendar-delete-multivalueextendedproperties.md)|None|Delete a multiValueExtendedProperties object.|
+|[Update multiValueExtendedProperties](../api/calendar-update-multivalueextendedproperties.md)|[multiValueLegacyExtendedProperty](../resources/multivaluelegacyextendedproperty.md)|Update the properties of a multiValueExtendedProperties object.|
+|[Get multiValueLegacyExtendedProperty](../api/multivaluelegacyextendedproperty-get.md)|[multiValueLegacyExtendedProperty](../resources/multivaluelegacyextendedproperty.md)|Read properties and relationships of a [multiValueLegacyExtendedProperty](../resources/multivaluelegacyextendedproperty.md) object.|
+|[List calendarPermissions](../api/calendar-list-calendarpermissions.md)|[calendarPermission](../resources/calendarpermission.md) collection|Get the calendarPermissions from the calendarPermissions navigation property.|
```

Comment:
Just so we're aware, I commented in some of the topics with the filename of the pattern calendar-{verb}-{nav property}.md that we may not have generated the relevant method topics for the calendar entity. For example, calendar-delete-events.md, calender-delete-calendarpermissions.md. Their HTTP request section doesn't look relevant.

Because this PR has 600+ topics, it's a bit challenging to get to specific topics ("Jump to" doesn't display that many files for us to select), I may have been exhaustive in identifying the problem.